proto/DataFormat.proto.m4

```diff
@@ -67,3 +67,23 @@ message DataSample {
   repeated SubseqSlot subseq_slots = 5;
 };
 
+message DataHeader2 {
+  enum SeqType {
+    NON_SEQ = 0;
+    SEQ = 1;
+    SUB_SEQ = 2;
+  }
+  repeated SlotDef slot_defs = 1;
+  repeated SeqType seq_type = 2;
+}
+
+message SlotSample {
```

Comment:
Need to comment how data is organized, whether missing slot is allowed, how INDEX data type is stored, etc.